src/callbacks/onPreprocess/flattenData/calculatePalt.js

```diff
@@ -0,0 +1,77 @@
+export default function calculatePalt(pt) {
+    // Calculates the pAlt value for the given participant ID
+    // For more on PAlt see the following paper: A Rapid Method to Estimate Hepatocyte Loss Due to Drug-Induced Liver Injury by Chung et al
+    // Requires: Baseline visit
+    // Assumes: Units on Alt are IU/L
+
+    var config = this.config;
+
+    //Get a list of raw post-baseline ALT values
+    var alt_values = pt.values.raw
+        .filter(f => f.analysisFlag)
```

Comment:
in examples 2 and 4 does this filter exclude the baseline result?

Reply:
It's independent from the baseline flag, so depends on the exact settings use case. 

I think I'm going to add a new `paltFlag` option instead though, since it sounds like the process for picking the values for this calculation is actually quite complex.